docs/examples/edit_distance.md

```diff
@@ -0,0 +1,121 @@
+# Edit Distance
+
+- Given two strings `str1` and `str2`, what is the minimum number of edits
+  to convert `str1` to `str2`?
+- Each edit costs 1, and is either an add, delete, or replace.
+- Goal: find a minimum cost set of edits that converts `str1` to `str2`.
+
+## Dynamic Programming Solution
+
+> A fully executable example can be found on our [GitHub](https://github.com/itsdawei/dpvis/tree/main/demos/edit_distance_2d.py).
+
+Let $OPT[i, j]$ denote the cheapest conversion between the first $i$ characters of `str1`
+and the first $j$ characters of `str2`.
+
+**BASE CASES: $i$ or $j$ is $0$.** At least one of our strings is empty, 
```

Comment:
For simplicity, can you break this down to two base cases (i=0 and j=0)? Thanks!